segments/path.bash

```diff
@@ -2,6 +2,7 @@
 _sbp_path_color_bg=${_sbp_path_color_bg-:$_sbp_color_blue}
 _sbp_path_color_fg=${_sbp_path_color_fg-:$_sbp_color_white}
 _sbp_path_color_sep=${_sbp_path_color_sep-:$_sbp_color_grey}
+_sbp_path_single_seg=${_sbp_path_single_seg-:1}
```

Comment:
Since this is a disabling of a feature I think we should go with something like _sbp_path_disable_sep, and if so. And in that case the default value should be 0, since default behaviour should be to enable the seperator. How does that sound to you?